guide/english/algorithms/algorithm-performance/index.md

```diff
@@ -3,20 +3,20 @@ title: Algorithm Performance
 ---
 
 ## Algorithm Performance
-To understand the performance of algorithms, first we need to know the big-O notation.
+To understand the performance of algorithms, first, we need to know the big-O notation.
 
 ### The Big-O Notation
-In mathematics, big-O notation is a symbolism used to describe and compare the _limiting behavior_ of a function.  
+In mathematics, the big-O notation is a symbolism used to describe and compare the _limiting behavior_ of a function.  
 
-A function's limiting behavior is how the function acts as it approaches a specific value (usually trends towards infinity).
+A function's limiting behavior is how the function acts as it approaches a specific value (usual trends towards infinity).
```

Comment:
Can you explain why you changed from `usually` to `usual`.  This does not make sense to me.
```
A function's limiting behavior is how the function acts as it approaches a specific value (usually trends towards infinity).
```